content/millennium/r4/documents/document-reference.md

```diff
@@ -71,13 +71,15 @@ _Implementation Notes_
 
 #### Request
 
-    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=12742400
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=1316024
```

Comment:
if we are changing patient id here, does it matter that on [line 85](https://github.com/cerner/fhir.cerner.com/pull/569/files#diff-d08dc65aa330ffe8d918982e3a661580b6666ecf31fac76d0a55223d8e4752f5R85)we still have the other patient id

Reply:
I updated the examples here: 52d2d270d86f726d2486f014f2534c21397ab663